viewer/packages/rendering/src/GeometryBatchingManager.ts

```diff
@@ -71,6 +76,12 @@ export class GeometryBatchingManager {
     this._sectorMap.delete(sectorId);
   }
 
+  private *getFloatsFromAttribute(attribute: THREE.InterleavedBufferAttribute): Iterable<number> {
+    for (let i = 0; i < attribute.count; i++) {
+      yield attribute.getX(i);
+    }
+  }
```

Comment:
This will be quite slow, iterators/generators are slow. I suggest you just use a plain loop instead below.

See https://jsbench.me/eyky1pj2ih/1 for a comparison on performance

Reply:
That sucks because the generator way of doing it is a lot more elegant imo. Didn't realize there was THAT much difference.

Reply:
I agree - code gets uglier, but it's just no way justifiable considering that this is in loading hotpath